helloworld-ios-app/HomeViewController.swift

```diff
@@ -38,18 +38,18 @@ class HomeViewController: UIViewController {
                 return
             }
             print("initialized OK")
-            self.button.hidden = false
+            self.button.isHidden = false
         }
     }
 
 
-    @IBAction func cloudCall(sender: AnyObject) {
+    @IBAction func cloudCall(_ sender: AnyObject) {
```

Comment:
Also, this `helloworld-ios-app/HomeViewController.HomeViewController.swift` should be `helloworld-ios-app/HomeViewController.swift`